content/docs/development/integration-tests.md

```diff
@@ -63,6 +63,30 @@ These tests can also be executed using docker:
   ```
 - clean up the etcd container: `docker rm -f dex-etcd`
 
+## Kubernetes
+
+Running integration tests for Kubernetes storage require the `DEX_KUBERNETES_CONFIG_PATH` environment variable
+with the path to kubeconfig file of the existing cluster. For tests, it is ok to use "mini" Kubernetes distributive, e.g., [KinD][kind], [Microk8s][microk8s].
```

Comment:
I'm not sure I understand this first sentence exactly. I think this fixes it, but am not sure if it is exactly true as per the actual config requirements:

```suggestion
Running integration tests for Kubernetes storage requires the `DEX_KUBERNETES_CONFIG_PATH` environment variable
be set with the path to kubeconfig file of the existing cluster. For tests, it is ok to use "mini" Kubernetes distributive, e.g., [KinD][kind], [Microk8s][microk8s].
```